qemu/tests/cfg/blockdev_snapshot_readonly.cfg

```diff
@@ -0,0 +1,28 @@
+- blockdev_snapshot_readonly:
+    type = blockdev_snapshot_readonly
+    virt_test_type = qemu
+    start_vm = yes
+    images += " data1"
+    remove_image_data1 = yes
+    force_create_image_data1 = yes
+    image_readonly_data1 = yes
+    storage_type_default = "directory"
+    storage_pool = default
+    image_size_data1 = 2G
+    image_format_data1 = qcow2
+    image_name_data1 = images/data1
+
+    snapshot_tag = sn1
+    image_format_sn1 = qcow2
+    image_name_sn1 = images/sn1
+    image_size_sn1 = ${image_size_data1}
+    image_readonly_sn1 = yes
+    device = "drive_data1"
+    base_tag = "data1"
+    rebase_mode = unsafe
+    error_msg = "Warning: Unable to open %s read-write (Read-only file system)"
```

Comment:
Just fyi.
I used to hit the issue that different msgs showed when configuring a read-only disk.

Reply:
Have updated it, please help to review again. zhenchao